sims/networks/basic/timing/GunnsRosesTiming.cpp

```diff
@@ -115,49 +121,59 @@ void GunnsRosesTiming::update(const double timestep)
                     N += 2;
                 }
             } else if (Gunns::GPU_DENSE == gpuMode) {
+                // Handle GPU DENSE mode
                 gpuTimePrev = gpuTime;
                 gpuTime = network->mSolveTimeAvg;
                 printf("    %9.2e", network->mSolveTimeAvg);
+                
                 if (errCheck) {
                     double error = 0.0;
-                    for (unsigned int i=0; i<5; ++i) {
-                        error += std::fabs(gpuPotentials[i] - cpuPotentials[i]);
```

Comment:
Please change to ```std::abs``` and resubmit.  We're trying to adhere to a new coding standard that explicitly uses the standard library namespace for all such math functions.